scvi/model/base/_base_model.py

```diff
@@ -481,6 +490,103 @@ def _check_if_trained(
             else:
                 raise RuntimeError(message)
 
+    def minify_adata(
```

Comment:
If the plan is to put this method under `BaseModelClass`, we should also move over the other methods under `BaseMinifiedModeModelClass` and then delete it. What do you think?